app/controllers/download_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Controller for downloading finding aid files
+class DownloadController < ApplicationController
+  def show
+    doc = SolrDocument.find(params[:id])
+
+    data_dir = ENV.fetch('DATA_DIR', Rails.root.join('data').to_s)
```

Comment:
For me this folder doesn't exist`Rails.root.join('data').to_s` so it had an unhandled error, but maybe that's fine.

Reply:
@marlo-longley I guess this is due to me typically using the `DownloadEadJob` to index finding aids and that task by default stores data in the `./data` directory. Sounds like you're usually using the fixture data. Maybe we should add the `./data` directory to the repo. I'm a little stumped about the right way to address this. Maybe a setting, but again I'm not sure if the default should be `./data` or `./spec/fixtures/ead`.

Reply:
Hm maybe we can have it all and make the default /data if it exists, then /spec/fixtures/ead?

Reply:
Sigh...I think switching to ActiveStorage to manage the files would solve this for us. Maybe I'll see where I can get with that.

Reply:
Ah I was not trying to cause a spiral with this comment! We could just commit the data directory or write this out for local dev instructions. Obviously on the deployed servers the ENV is set and the directory exists.

Reply:
@marlo-longley made a couple adjustments, including making the fixtures directory the default for development for now.